### DISCUSSION TITLE
Visual Studio IronScheme Integration

### DISCUSSION THREAD

**Discussion by RudraveerMandal** [leppie/IronScheme#134] [Discussions]
I couldn't get it to work. How exactly do I do it? Is it still supported?

**Comment by leppie** [comment]
Did you install this? https://github.com/IronScheme/IronScheme.VisualStudio2/releases/tag/1.999.0.49

**Comment by leppie** [comment]
I found a bug with VS2022, 

New release https://github.com/IronScheme/IronScheme.VisualStudio2/releases/tag/1.999.0.55
